war/src/main/less/styles.less

```diff
@@ -8,7 +8,7 @@ html {
   // The downside is that the page does not resize with the browser's font size,
   // only with the zoom level.
   font-size: 16px !important;
-  letter-spacing: -0.011em;
+  letter-spacing: -0.016em;
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing

> Legible letter-spacing must be determined on a case-by-case basis, as different font families have different character widths. There is no one value that can ensure all font families automatically maintain their legibility.

Is that not a concern here? If we're just using whatever is the system font, won't that mean we have no way of knowing whether messing with letter spacing will work out?